index.html

```diff
@@ -54,164 +62,153 @@ <h2>LET'S CONNECT</h2>
-        </ul>
-        <button class="btn-card">See project</button>
-        </div>
-      </section>
+                </article>
+                <!--third card-->
+                <article class="card">
 
-      <section class="container-work2">
-        <div class="card-info">
-        <div class="card-image">
-          <img src="images/SnapshootPortfolio.png" alt="">
-        </div>
-        <div class="card-header">
-          <h2>Tonic</h2>
-        </div>
-        <ul class="card-details">
-          <li class="card-first-element">CANOPY</li>
-          <li class="circle-icon"><i class="fa fa-circle"></i></li>
-          <li class="card-second-element">Back End Dev</li>
-          <li class="circle-icon"><i class="fa fa-circle"></i></li>
-          <li class="card-third-element">2015</li> 
-          </ul>
-        <div class="card-content">
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        </div>
-        <ul class="btn-button">
-          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
-          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
-          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
-        </ul>
-        <button class="btn-card">See project</button>
-        </div>
-      </section>
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-3.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>multi-post stories</h3>
+                    <h4><span>Facebook</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      full stack dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>Exploring the future of media in facebook's first virtual Reality <br>
+                      app a place to discover and enjoy 360 points and videos on <br>
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>Ruby on rails</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+              </section>
+              <section class="about">
+                <div class="about-container">
+                  <div class="about-content">
+                  <h2>About <br>Myself</h2>
+                  <p>Hello I’m a software developer! I can help you build a product ,<br> feature
+                    or website Look through some of my work and experience!<br> If you like what
+                    you see and have a project you need coded, <br> don’t hestiate to contact me.</p>
```

Comment:
- [ ] I noticed that your text color is black for both the `<h2>` and `<p>` tags. Please use the specified colors. The goal is to learn how to follow design guidelines and specifications as closely as possible.
For the `<h2>` tag, use `#091E42` and for the `<p>`, please use `#344563` 🙏